Gemfile

```diff
@@ -1,5 +1,5 @@
-source "http://rubygems.org"
+source "https://rubygems.org"
 
-# Specify your gem's dependencies in resque-pool.gemspec
 gemspec
-gem "json" # so 1.8 can have a halfway decent json gem
+
+gem 'mustache', '~> 0.99.8' # 1.0 needs ruby 2.0
```

Comment:
mustache 1.0 did not work on 1.9 ... but I kicked that out ... so it can go
away ... followup PR coming

On Tue, Oct 6, 2015 at 8:14 AM, Joshua Flanagan <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/nevans/resque-pool/pull/129#discussion_r41277405>:
>
> >  gemspec
> > -gem "json" # so 1.8 can have a halfway decent json gem
> > +
> > +gem 'mustache', '~> 0.99.8' # 1.0 needs ruby 2.0
>
> Why was mustache added to the Gemfile? Seems odd...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nevans/resque-pool/pull/129/files#r41277405>.
>